packages/types/src/lib/interfaces.ts

```diff
@@ -1094,13 +1127,13 @@ export interface CommonGetSessionSigsProps {
   /**
    * @deprecated - use capabilityAuthSigs instead
    *  Used for delegation of Capacity Credit. This signature will be checked for proof of capacity credit.
-   * on both manzano and habanero networks capacity credit proof is required.
-   * see more here: https://developer.litprotocol.com/v3/sdk/capacity-credits
+   * on both Datil-test and Datil networks capacity credit proof is required.
```

Comment:
Im not sure if we want to use specific network names here as we will have to keep updating this comment header

Reply:
Changed to:
> Capacity credits are required on the paid Lit networks (mainnets and certain testnets), and are not required on the unpaid Lit networks (certain testnets).

Tried to make it as general as possible, I'm not sure how to specify them otherwise without mentioning the networks by name.

Reply:
Updated the below comment to say:
>When the session key signs the SIWE ReCap message, this will be the session key pair public key. If an EOA wallet signs the message, then this will be the EOA Ethereum address.